packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -61,27 +65,39 @@ type DismissProps =
       };
 
 /**
- * Provide a specific HTML tag that overrides the default (`div`).
- *
- * Notes:
- * - When `tag="section"` or `"figure"`, `cardAriaLabel` is required for accessibility.
- * - `button` and `a` tags are not allowed - use Wonder Blocks Button and Link components as children instead.
  * Valid HTML tags for the Card component.
  * Excludes button and anchor tags which should use Wonder Blocks Button and Link components instead.
  */
-type ValidCardTags = Exclude<keyof JSX.IntrinsicElements, "button" | "a">;
 
-type TagProps =
-    | {
-          // Section and figure require an aria-label
-          tag: "section" | "figure";
-          labels: {cardAriaLabel: string} & Record<string, any>;
-      }
-    | {
-          // All other valid tags except button and a
-          tag?: Exclude<ValidCardTags, "section" | "figure">;
-          labels?: Record<string, any>;
-      };
+/**
+ * Accessibility props for the Card.
+ *
+ * Labeling methods (in order of preference):
+ * 1. `labels.cardAriaLabel` - For translatable strings (preferred)
+ * 2. `aria-labelledby` - For ID references
+ * 3. `aria-label` - also allowed as a fallback. `labels.cardAriaLabel` automatically
+ * applies this attribute.
+ *
+ * Multiple methods can be provided for consumer simplicity, but only one will win
+ * based on standard Accessible Name Computation rules.
+ */
+type AccessibilityProps = {
+    labels?: {
+        cardAriaLabel?: string;
+        dismissButtonAriaLabel?: string;
+    };
+    "aria-labelledby"?: string;
+    "aria-label"?: string;
+    "aria-busy"?: AriaAttributes["aria-busy"];
+    "aria-roledescription"?: AriaAttributes["aria-roledescription"];
```

Comment:
Should the Card component props extend the AriaProps? This is normally what we do in components so it supports all the aria attributes! 

Also, if we want to include support for setting the `role`, AriaProps includes the type for `role`, while AriaAttributes doesn't!

Reply:
I personally don't think it's a good practice to support all ARIA attributes everywhere because it can allow non-compliant markup. We need a way to restrict some of them to valid use cases, so opting in for common usage through direct props makes sense to me. We can always enable support for something specific if and when it comes up!

Reply:
That's fair! I think this makes sense to start doing going forwards. cc: @jandrade also in case you have thoughts on this!

Another nice thing about this is when we explicitly add the specific aria props, it'll get included in the prop docs automatically! (props extended from the AriaProps weren't automatically included the props docs)